assets/js/state/databases.js

```diff
@@ -30,6 +30,11 @@ export const databasesListSlice = createSlice({
     appendDatabaseInstance: (state, action) => {
       state.databaseInstances = [...state.databaseInstances, action.payload];
     },
+    removeDatabase: (state, { payload: { id } }) => {
+      state.databases = state.databases.filter(
+        (database) => !(database.id === id)
```

Comment:
is there a reason to use `!(database.id === id)` over `database.id !== id` here?